### PR TITLE
r? ianbicking Fixes #257 Cache busting query strings

### LINKED ISSUE #257: Cache static files and put cache-busting query string in links to them

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: server/src/server.js [hunk 1]
```diff
@@ -110,6 +114,7 @@ let server = http.createServer(function (req, res) {
         params: state.params,
         query: state.query}
     ).then(function (data) {
+      data.git_revision = git_revision;
```
Comment (ianb) [inline server/src/server.js]: camelCaseIt!

Comment (fzzzy) [inline server/src/server.js]: Whoops!

File: server/src/views/shell.js [hunk 2]
```diff
@@ -9,9 +9,9 @@ exports.Shell = React.createClass({
         <title>
           PageShot
         </title>
-        <script src="/js/bundle.js" />
-        <link rel="stylesheet" href="/css/styles.css" />
-        <link rel="stylesheet" href="/css/login.css" />
+        <script src={"/js/bundle.js?git_revision=" + this.props.git_revision } />
+        <link rel="stylesheet" href={"/css/styles.css?git_revision=" + this.props.git_revision } />
+        <link rel="stylesheet" href={"/css/login.css?git_revision=" + this.props.git_revision }/>
```
Comment (ianb) [inline server/src/views/shell.js]: Maybe we can pass something like `props.linkify=function (link) {return link +(link.contains("?") ? "?" : "&") + "bust=" + gitRevision}` and then avoid the manual link construction here.

**Comment by ianb** [comment]
Pushed, assuming the nits can get picked up later

File: server/src/views/shell.js (hunk 2 above)
Comment (fzzzy) [inline server/src/views/shell.js]: Sounds good

**Pull request merged**